packages/create-block/lib/scaffold.js

```diff
@@ -99,6 +100,8 @@ module.exports = async (
 		editorScript,
 		editorStyle,
 		style,
+		...getTemplateVariantVars( variants, variant ),
+		...variants[ variant ],
```

Comment:
That setup will probably override the input provided by the user. It might be a little bit tricky to propagate the default values for the variant selected correctly. Actually, if you would force the user to pick the variant for the block first, then you could tweak the default values passed to prompts and that would cover everything.

Reply:
I was right. See the screenshot:

<img width="1181" alt="Screenshot 2022-08-19 at 14 19 41" src="https://user-images.githubusercontent.com/699132/185636599-b0bc45a8-9c25-404f-909c-d670c2b2e165.png">

I selected the default value hinted in the prompt. It wasn't the one coming from the variant. However, at the end of the scaffolding, the values from the variant were used instead and that's why the folder name is different (on the right side in the explored).

Reply:
@gziolo I believe I have addressed this. If you could test it again when have have a chance. Thanks!